doc/sphinx/user/methods/nonlinear-stokes-solvers.md

```diff
@@ -197,7 +196,7 @@ Some strategies to achieve this, also called globalization, will be discussed in
 
 
 (user_methods_nonlinear_solvers_sin_1)=
-```{tikz} Solving $sin(x)=0$ through with the Newton method, with starting guess $x=1$ and the derivative of that $x$ plotted.
+```{tikz} Solving $sin(x)=0$ through with the Newton method with starting guess $x=1$ and its the derivative is plotted.
```

Comment:
```suggestion
```{tikz} Solving $sin(x)=0$ through the Newton method with starting guess $x=1$ and its derivative is plotted.
```